shell/browser/api/save_page_handler.cc

```diff
@@ -30,6 +30,12 @@ void SavePageHandler::OnDownloadCreated(content::DownloadManager* manager,
 
 bool SavePageHandler::Handle(const base::FilePath& full_path,
                              const content::SavePageType& save_type) {
+  if (!full_path.IsAbsolute()) {
```

Comment:
It is better to have this check in `api::WebContents::SavePage` as an early termination which avoids creating `SavePageHandler` in that case.